governance/program/src/state/proposal.rs

```diff
@@ -1078,13 +1093,22 @@ pub fn get_proposal_address<'a>(
     .0
 }
 
-/// Assert options to create proposal are valid for the Proposal vote_type
-pub fn assert_valid_proposal_options(
-    options: &[String],
+/// Assert options to create proposal are valid
+pub fn assert_valid_proposal_options(options: &[String]) -> Result<(), ProgramError> {
+    if options.iter().any(|o| o.is_empty()) {
+        return Err(GovernanceError::InvalidProposalOptions.into());
+    }
+
+    Ok(())
+}
+
+/// Assert options to assign proposal are valid for Proposal vote_type
+pub fn assert_valid_proposal_options_on_signoff(
```

Comment:
is this still needed if we check the options in every `AddOptions` instruction?

Reply:
The check is split to two parts now.
There is a check if option description is empty for `AddProposalOptions`  ix and then a check if the list of options are empty during sign off. To me it makes sense to have that separation but we can change it.